weave/Scripts/Main.cs

```diff
@@ -50,6 +53,16 @@ public override void _Ready()
         SetupLogger();
     }
 
+    public override void _Process(double delta)
+    {
+        _players.ForEach(
```

Comment:
Should discuss this before merging ☺️

Reply:
You're thinking about good values? Or what its based on

Reply:
Perhaps both

Reply:
😊